src/utils.js

```diff
@@ -127,7 +127,7 @@ type Group = {|
  * cases, this will just be the group itself, but when ordgroups and colors have
  * a single element, we want to pull that out.
  */
-const getBaseElem = function(group: Group) {
+const getBaseElem = function(group: Group): Group | boolean {
```

Comment:
(optional) Just a side node: I think it's more canonical to return `?Group` rather than `Group | boolean`.

Reply:
`?Group` would allow the return value to be `null`, but not `false`.

Reply:
That's exactly what I meant (sorry, I wasn't clear). This would require changing the return value to `null` and changing where it's used. That's more in line with what we do elsewhere -- if we're conditionally returning a value, we typically use a nullable.

Reply:
Sorry for the misunderstanding.  I'll change that in a followup PR.